binary_transparency/firmware/cmd/ft_witness/impl/ft_witness.go

```diff
@@ -0,0 +1,74 @@
+// Copyright 2020 Google LLC. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Package impl is the implementation of the Firmware Transparency witness server.
+// This requires a Trillian instance to be reachable via gRPC and a tree to have
+// been provisioned.
```

Comment:
This comment about Trillian and trees seems a bit odd. It's technically true, but at this layer in the stack we should just be talking about an FT Log running at a known address.

Reply:
Done!